parallel_algorithms/P3179/P3179.bs

```diff
@@ -171,18 +171,18 @@ to the C++17 parallel algorithms are proposed:
 ### An example of the proposed API ### {#design_api_example}
 
 ```cpp
-template<class ExecutionPolicy, random_access_iterator I, sized_sentinel_for<I> S,
+template<execution-policy ExecutionPolicy, random_access_iterator I, sized_sentinel_for<I> S,
          random_access_iterator O, sized_sentinel_for<O> OutS,
          copy_constructible F, class Proj = identity>
   requires indirectly_writable<O, indirect_result_t<F&, projected<I, Proj>>>
   ranges::unary_transform_result<I, O>
     ranges::transform(ExecutionPolicy&& exec, I first, S last, O result, OutS result_last,
                       F op, Proj proj = {});
 
-template<class ExecutionPolicy, ranges::random_access_range R, ranges::random_access_range OutR,
+template<execution-policy ExecutionPolicy, ranges::random_access_range R, ranges::random_access_range OutR,
          copy_constructible F, class Proj = identity>
   requires indirectly_writable<ranges::iterator_t<OutR>, indirect_result_t<F&, projected<ranges::iterator_t<R>, Proj>>>
-           && ranges::sized_range<R> && ranges::sized_range<OutR>
+           && sized_range<R> && sized_range<OutR>
```

Comment:
I do not understand this change, since other library names in the signature still keep the `ranges::` prefix.

Reply:
Discussed offline. Todo for R5